jms-core/src/main/java/io/micronaut/jms/listener/ConcurrentMessageHandler.java

```diff
@@ -29,7 +29,9 @@
  * @param <T> the type of the object that the handler is expecting.
  * @author Elliott Pope
  * @since 1.0.0
+ * @deprecated since 1.0.0.M2. All associated logic is handled by the {@link JMSListener}.
```

Comment:
Go ahead and delete, not much need to be backwards-compatible until official 1.0.0